src/main/kotlin/snyk/common/CustomEndpoints.kt

```diff
@@ -1,14 +1,16 @@
 package snyk.common
 
 import io.snyk.plugin.pluginSettings
-import io.snyk.plugin.prefixIfNot
```

Comment:
This will be re-added. Have uncommitted commeted code that made `prefixIfnot` unused